honeybee_radiance_command/options/rpict.py

```diff
@@ -0,0 +1,548 @@
+from .optionbase import (
+    OptionCollection,
+    BoolOption,
+    NumericOption,
+    StringOption,
+    StringOptionJoined,
+    IntegerOption,
+    TupleOption,
+)
+
+
+class RpictOptions(OptionCollection):
+    """rpict command options."""
+
+    __slots__ = (
+        "_vt",
+        "_vp",
+        "_vd",
+        "_vu",
+        "_vh",
+        "_vv",
+        "_vo",
+        "_va",
+        "_vs",
+        "_vl",
+        "_x",
+        "_y",
+        "_pa",
+        "_pj",
+        "_pm",
+        "_pd",
+        "_ps",
+        "_pt",
+        "_t",
+        "_w",
+        "_i",
+        "_u",
+        "_bv",
+        "_dt",
+        "_dc",
+        "_dj",
+        "_ds",
+        "_dr",
+        "_dp",
+        "_dv",
+        "_ss",
+        "_st",
+        "_av",
+        "_aw",
+        "_ab",
+        "_aa",
+        "_ar",
+        "_ad",
+        "_as_",
+        "_me",
+        "_ma",
+        "_mg",
+        "_ms",
+        "_lr",
+        "_lw",
+        "_am",
+    )
+
+    def __init__(self):
+        """rpict command options."""
+        self._vt = StringOptionJoined("vt", "view type perspective - default: vtv")
```

Comment:
for these options you have to provide valid values otherwise users can do whatever they want. See here:

https://github.com/ladybug-tools/honeybee-radiance-command/blob/c9b1c59fb1c17dcc84f3e1a9fc144acc8309fa43/honeybee_radiance_command/options/rtrace.py#L42-L48